techdebt: [CDS-82834]: Revamp Connect and Manage Argo Providers - technical changes (#4336)
harness/developer-hub: docs/continuous-delivery/gitops/use-gitops/install-a-harness-git-ops-agent.md

Select **New GitOps Agent**. The Agent wizard appears.

:::note

> For the purposes of this tutorial, we will be creating a regular Harness GitOps agent without an existing Argo CD instance (non-BYOA setup). For creating a BYOA setup (with an existing Argo CD installation), please refer to [Bring Your Own Argo CD (BYOA) into Harness GitOps](/docs/continuous-delivery/gitops/connect-and-manage/multiple-argo-to-single-harness.md)

Reviewer: ```suggestion
For the example in this topic, a regular Harness GitOps agent is created without an existing Argo CD instance (non-BYOA setup). For creating a BYOA setup (with an existing Argo CD installation), please refer to [Bring Your Own Argo CD (BYOA) into Harness GitOps](/docs/continuous-delivery/gitops/connect-and-manage/multiple-argo-to-single-harness)
```

Reviewer: exclude the .md in the link. our site generator can have issues with it. If you try use .md in a URL in our site, it fails. try it and see.
